perllib/Utils.pm

```diff
@@ -12,7 +12,6 @@ package Utils;
 use strict;
 use DateTime;
 use Encode;
-use File::Slurp qw();
```

Comment:
I guess we can remove this from the `cpanfile` as well :)

Reply:
I wondered about that, but I also wondered whether to remove it from `cpanfile.snapshot` as well, then wondered whether unpinning its version from that file might cause problems.  In the end I decided not to overthink it and take the safe, sub-optimal approach.  If you're happy making that further removal, I cautiously approve though.

Reply:
It would cause problems if removed from the snapshot, yes. `cpanfile` is the list of modules used by the project (similar to e.g. a Gemfile); `cpanfile.snapshot` is a cache of the exact versions installed of all modules and their dependencies (like Gemfile.lock). So removing it from `cpanfile` (as we're not using it any more ourselves) is fine but wouldn't change the snapshot, because other modules that we do use still depend upon it.

Reply:
Thanks for the explanation.  I've worked with carton and cpanfile before, but the details never stick in my head as well as I'd hope.